src/test/java/io/odh/test/e2e/standard/ModelServingST.java

```diff
@@ -0,0 +1,329 @@
+                .addToStringData("aws-connection-no-such-connection", "{}")
+                .build();
+        ResourceManager.getInstance().createResourceWithWait(storageConfig);
+
+        // create serving runtime
+        ServingRuntime servingRuntime = processModelServerTemplate("ovms");
+        ServingRuntime servingRuntimeInstance = new ServingRuntimeBuilder(servingRuntime)
+                .editMetadata()
+                .withName(runtimeName)
+                .withNamespace(projectName)
+                .addToAnnotations("enable-route", "true")
+                .addToAnnotations("opendatahub.io/apiProtocol", "REST")
+                .addToAnnotations("opendatahub.io/accelerator-name", "")
+                .addToLabels(OdhAnnotationsLabels.LABEL_DASHBOARD, "true")
+                .endMetadata()
+                .editSpec()
+                .editFirstContainer()
+                // with unspecified resources, one would get
+                //  java.lang.Exception: Reported model capacity -0.125GiB too small relative to advertised default model size of 1MiB
+                .editContainersResources()
+                .addToLimits("cpu", new IntOrString("2"))
+                .addToLimits("memory", new IntOrString("8Gi"))
+                .addToRequests("cpu", new IntOrString("1"))
+                .addToRequests("memory", new IntOrString("4Gi"))
+                .endContainersResources()
+                .addToVolumeMounts(new VolumeMountsBuilder().withMountPath("/dev/shm").withName("shm").build())
+                .endServingruntimespecContainer()
+                .addToVolumes(new VolumesBuilder().withName("shm").withEmptyDir(new EmptyDirBuilder().withMedium("Memory").withSizeLimit(new IntOrString("2Gi")).build()).build())
+                .endSpec()
+                .build();
+        ResourceManager.getInstance().createResourceWithWait(servingRuntimeInstance);
+
+        // create inference service
+        InferenceService inferenceService = new InferenceServiceBuilder()
+                .withNewMetadata()
+                .withName(modelName)
+                .withNamespace(projectName)
+                .addToLabels("opendatahub.io/dashboard", "true")
+                .addToAnnotations("openshift.io/display-name", modelName)
+                .addToAnnotations("serving.kserve.io/deploymentMode", "ModelMesh")
+                .endMetadata()
+                .withNewSpec()
+                .withNewPredictor()
+                .withNewModel()
+                .withNewModelFormat()
+                .withName("onnx")
+                .withVersion("1")
+                .endModelFormat()
+                .withRuntime(runtimeName)
+                // https://github.com/onnx/models/blob/main/validated/vision/classification/mnist/README.md
+                .withStorageUri("https://github.com/onnx/models/blob/bec48b6a70e5e9042c0badbaafefe4454e072d08/validated/vision/classification/mnist/model/mnist-8.onnx?raw=true")
```

Comment:
maybe would make sense to have this as a constant somewhere at the beginning of the file so it's more visible

Reply:
done